easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -96,6 +96,9 @@ def configure_step(self, srcdir=None, builddir=None):
             if value is not None:
                 options.append("-D%s='%s'" % (option, value))
 
+        # show what CMake is doing by default
+        options.append("-DCMAKE_VERBOSE_MAKEFILE=ON")
```

Comment:
well, bigger logs ;)

but you're right, I see no reason not to include this

others have been doing this for ages, cfr. https://bugzilla.redhat.com/show_bug.cgi?id=474053